deps/rabbit/src/rabbit_quorum_queue.erl

```diff
@@ -406,7 +406,14 @@ handle_tick(QName,
                                {messages_persistent, M}
 
                                | infos(QName, ?STATISTICS_KEYS -- [consumers])],
-                      rabbit_core_metrics:queue_stats(QName, Infos),
+                      {ok, Queue} = rabbit_amqqueue:lookup(QName),
+                      Up = case ra:consistent_query(amqqueue:get_pid(Queue), fun (_) -> ok end) of
```

Comment:
This can be really expensive to do this for every quorum queue every N seconds with 10s of thousands of queues.

Reply:
Yes, see my comment in the PR under `Open questions`. We can execute it less often. If it's still too expensive, can you think of another way to report a metric for quorum queue availability?

Reply:
I think your problems may come from the use of get_pid here. As this executes on the (assumed) leader node you should just be able to use 'Name' directly to issue a request to the leader on the local node. (Assuming Name is the locally registered name of the QQ process, else we'd have to pass that along).

Reply:
The consistent query timeout should probably be less than the tick interval btw.

Reply:
Using `?RPC_TIMEOUT` now since that is used by other RPCs as well. Might be too low however. I was thinking of using something like `round(0.75 * ?TICK_TIMEOUT)`.